Marketplace/voiceHUDApp/App Resources/appData/resources/css/styles.css

```diff
@@ -0,0 +1,46 @@
+body {
+    margin: -10px 50px 50px 50px;
+    width: 100% - 10px;
+    font-size: 20px;
+    font-family: 'Raleway', sans-serif;
+    font-weight: bold;
+    color: white;
+    background: linear-gradient(#ffffff, #000000);
+    background-image: url("https://hifi-content.s3.amazonaws.com/brosche/dev/voiceHUDApp/App%20Resources/appData/resources/images/background.png");
+    background-size: cover;
+}
+
+#info {
+    font-size: 25px;
+    background-color: rgb(64, 64, 64);
+}
+
+.blue {
+    color: rgba(0,255,255,1);
+    font-family: 'Raleway';
```

Comment:
The class name doesn't make as much sense with the other stuff going on here.  Also the rgb value's aren't exactly blue.  maybe fix the classname?